rasa_nlu/training_data/formats/markdown.py

```diff
@@ -1,15 +1,13 @@
 import logging
 import re
-import typing
-from typing import Any, Text
+from typing import Any, Text, TYPE_CHECKING
 
 from rasa_nlu.training_data.formats.readerwriter import (
     TrainingDataReader,
     TrainingDataWriter)
-from rasa_nlu.training_data.util import check_duplicate_synonym
 from rasa_nlu.utils import build_entity
 
-if typing.TYPE_CHECKING:
```

Comment:
this shouldn't change, first because we use `typing.TYPE_CHECKING` everywhere else and because we should actually always access variables / functions through the module name and not import them directly

Reply:
:+1: